api/FileReader.json

```
@@ -608,7 +608,8 @@
               "version_added": "3.6"
             },
             "firefox_android": {
-              "version_added": "32"
+              "version_added": "32",
+              "notes": "Using the camera in Android 8.x raises an exception."
```

As a conveninece, I think we ought to link to the corresponding bug here, like this:

```suggestion
              "notes": "Using the camera in Android 8.x raises an exception. See <a href='https://bugzil.la/1511083'>bug 1511083</a>."
```

---

Oh yeah, I'm dumb, especially considering I *commented* on that very bug!

---

Not dumb, hardly anything's perfect on the first iteration :-)